src/azure-cli/azure/cli/command_modules/cdn/_help.py

```diff
@@ -40,6 +40,10 @@
     text: >
         az cdn custom-domain create -g group --endpoint-name endpoint --profile-name profile \\
             -n domain-name --hostname www.example.com
+  - name: Enable custom https with a minimum
+    text: >
+        az cdn custom-domain create -g group --endpoint-name endpoint --profile-name profile \\
+            -n domain-name --hostname www.example.com
```

Comment:
a single `\` will appear in the line. It's YAML restriced.
If you want to make the example stay in a line, pelease remove the `\\`.
If you want to the examples wrap like you want, please use `text: |` instead of `>`

Reply:
mamy case below.

Reply:
I decided to remove the back slashes altogether since not all shells use backslash as a line continuation character. Thanks for the input.